apps/communications/contacts/views/details/js/boot.js

```diff
@@ -0,0 +1,58 @@
+/* global DetailsUI, Details, LazyLoader, fbLoader, ContactsService */
+'use strict';
+
+window.addEventListener('DOMContentLoaded', function() {
+  LazyLoader.load([
+    '/shared/js/l10n.js'], function() {
+    LazyLoader.load([
+      document.getElementById('view-contact-details')
+    ], function() {
+      // TODO Add if needed
+    });
+  });
+});
+
+window.onload = function() {
+  var dependencies = [
+    '/contacts/style/webrtc-client/webrtc_client.css',
+    '/contacts/js/webrtc-client/webrtc_client.js',
+    '/contacts/services/contacts.js',
+    '/contacts/js/activities.js',
+    '/shared/js/contacts/utilities/event_listeners.js',
```

Comment:
We can remove this as well. No need to load an entire JS class to add a listener. Modify https://github.com/mozilla-b2g/gaia/pull/30750/files#diff-53dea38d3841fcc8551cbbb553fb4171R44 to add listeners directly, and remove the dependency.